corehq/apps/sso/forms.py

```diff
@@ -915,8 +910,7 @@ def __init__(self, identity_provider, *args, **kwargs):
                 css_class="panel panel-modern-gray panel-form-only"
             ),
         )
-        if self.show_remote_user_management:
-            layout.append(crispy.Div(*self.get_remote_user_management_fields()))
+        layout.append(crispy.Div(*self.get_remote_user_management_fields()))
         layout.append(crispy.Div(*self.get_primary_fields()))
 
         self.helper.layout = layout
```

Comment:
`self.helper.layout` can also be moved back to line where `layout` is assigned, as `layout.append` is no longer needed